logistic_order_multicurrency/i18n/logistic_order_multicurrency.pot

```diff
@@ -0,0 +1,22 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* logistic_order_multicurrency
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-01-08 14:03+0000\n"
+"PO-Revision-Date: 2015-01-08 14:03+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: logistic_order_multicurrency
+#: model:ir.model,name:logistic_order_multicurrency.model_sale_order
+msgid "Sales Order"
+msgstr ""
+
```

Comment:
This file looks out of date (there is no mention of the added fields in the module).